[Automation] javac compile fails for org.jboss.weld:weld-spi:2.0.Final

## javac compile on ubuntu-latest / JDK 25 / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: 25
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.0.Final" ./gradlew clean compileTestJava -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25260953949/job/74068131062
Last 300 lines of the log:
```console
    required: String
    found:    String,String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isTrue();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
          singleton.clear(singletonId);
                   ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
13 actionable tasks: 5 executed, 8 up-to-date
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  7 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 5s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.jboss.weld:weld-spi:1.2.0.Beta1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 13s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 2.0.Final failed with exit code 1.
SOFT-FAIL[native-image run][2.0.Final][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.0.Final" ./gradlew clean test -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"]
========================================================================================
 org.jboss.weld:weld-spi:1.2.0.Beta1:2.0.Final stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 911ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE

/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:38: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isFalse();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:39: error: method get in interface Singleton<T> cannot be applied to given types;
> Task :compileTestJava FAILED
        assertThatThrownBy(() -> singleton.get(singletonId))
                                          ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
        singleton.set(singletonId, "stored value");
                 ^
  required: String
  found:    String,String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isTrue();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
        singleton.clear(singletonId);
                 ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isFalse();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
7 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:38: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:39: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThatThrownBy(() -> singleton.get(singletonId))
                                            ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
          singleton.set(singletonId, "stored value");
                   ^
    required: String
    found:    String,String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isTrue();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.get(singletonId)).isEqualTo("stored value");

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
10 actionable tasks: 1 executed, 9 up-to-date
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
          singleton.clear(singletonId);
                   ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  7 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'javac compile' for 2.0.Final failed with exit code 1.
FAILED[javac compile][2.0.Final][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.0.Final" ./gradlew clean compileTestJava -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode current-defaults

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: current-defaults
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.0.Final" ./gradlew clean compileTestJava -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25260953949/job/74068131064
Last 300 lines of the log:
```console
                                            ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
          singleton.set(singletonId, "stored value");
                   ^
    required: String
    found:    String,String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isTrue();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
          singleton.clear(singletonId);
                   ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  7 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 4s

> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.jboss.weld:weld-spi:1.2.0.Beta1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 14s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 2.0.Final failed with exit code 1.
SOFT-FAIL[native-image run][2.0.Final][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.0.Final" ./gradlew clean test -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"]
========================================================================================
 org.jboss.weld:weld-spi:1.2.0.Beta1:2.0.Final stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 939ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:38: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isFalse();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton

> Task :compileTestJava FAILED
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:39: error: method get in interface Singleton<T> cannot be applied to given types;
        assertThatThrownBy(() -> singleton.get(singletonId))
                                          ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
        singleton.set(singletonId, "stored value");
                 ^
  required: String
  found:    String,String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isTrue();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
        singleton.clear(singletonId);
                 ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isFalse();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
7 errors

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:38: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:39: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThatThrownBy(() -> singleton.get(singletonId))
                                            ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
          singleton.set(singletonId, "stored value");
                   ^
    required: String
    found:    String,String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isTrue();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
          singleton.clear(singletonId);
                   ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
10 actionable tasks: 1 executed, 9 up-to-date

          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  7 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'javac compile' for 2.0.Final failed with exit code 1.
FAILED[javac compile][2.0.Final][GVM_TCK_NATIVE_IMAGE_MODE="current-defaults" GVM_TCK_LV="2.0.Final" ./gradlew clean compileTestJava -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"]
```

## javac compile on ubuntu-latest / JDK latest-ea / mode future-defaults-all

Failure kind: javac compile
OS: ubuntu-latest
JDK: latest-ea
Native-image mode: future-defaults-all
Reproducer: `GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.0.Final" ./gradlew clean compileTestJava -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"`
Runner log: https://github.com/oracle/graalvm-reachability-metadata/actions/runs/25260953949/job/74068131040
Last 300 lines of the log:
```console
          singleton.set(singletonId, "stored value");
                   ^
    required: String
    found:    String,String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:

      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isTrue();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
          singleton.clear(singletonId);
                   ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  7 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).
Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
13 actionable tasks: 5 executed, 8 up-to-date
BUILD FAILED in 4s


> Task :tckTest FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':tckTest'.
> Test for org.jboss.weld:weld-spi:1.2.0.Beta1 failed with exit code 1.

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 13s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'native-image run' for 2.0.Final failed with exit code 1.
SOFT-FAIL[native-image run][2.0.Final][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.0.Final" ./gradlew clean test -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"]
========================================================================================
 org.jboss.weld:weld-spi:1.2.0.Beta1:2.0.Final stage javac compile
========================================================================================
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE

> Configure project :
GraalVM Reachability Metadata TCK
---------------------------------

> Task :tckClean
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :clean

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD SUCCESSFUL in 897ms
9 actionable tasks: 1 executed, 8 up-to-date
Consider enabling configuration cache to speed up this build: https://docs.gradle.org/9.1.0/userguide/configuration_cache_enabling.html

> Task :clean

> Task :compileTestJava FAILED
> Task :tck-build-logic:extractPluginRequests UP-TO-DATE
> Task :tck-build-logic:generatePluginAdapters UP-TO-DATE
> Task :tck-build-logic:compileJava UP-TO-DATE
> Task :tck-build-logic:compileGroovy UP-TO-DATE
> Task :tck-build-logic:compileGroovyPlugins UP-TO-DATE
> Task :tck-build-logic:pluginDescriptors UP-TO-DATE
> Task :tck-build-logic:processResources UP-TO-DATE
> Task :tck-build-logic:classes UP-TO-DATE
> Task :tck-build-logic:jar UP-TO-DATE
> Task :tck-test-support:compileJava UP-TO-DATE
> Task :compileJava NO-SOURCE
> Task :processResources NO-SOURCE
> Task :classes UP-TO-DATE
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:38: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isFalse();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:39: error: method get in interface Singleton<T> cannot be applied to given types;
        assertThatThrownBy(() -> singleton.get(singletonId))
                                          ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
        singleton.set(singletonId, "stored value");
                 ^
  required: String
  found:    String,String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton

> Task :compileTestJava FAILED

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/build/reports/problems/problems-report.html
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isTrue();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
        singleton.clear(singletonId);
                 ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
/home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
        assertThat(singleton.isSet(singletonId)).isFalse();
                            ^
  required: no arguments
  found:    String
  reason: actual and formal argument lists differ in length
  where T is a type-variable:
    T extends Object declared in interface Singleton
7 errors

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed; see the compiler output below.
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:38: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:39: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThatThrownBy(() -> singleton.get(singletonId))
                                            ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:43: error: method set in interface Singleton<T> cannot be applied to given types;
          singleton.set(singletonId, "stored value");
                   ^
    required: String
    found:    String,String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:44: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isTrue();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:45: error: method get in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.get(singletonId)).isEqualTo("stored value");
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:47: error: method clear in interface Singleton<T> cannot be applied to given types;
          singleton.clear(singletonId);
                   ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  /home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java:48: error: method isSet in interface Singleton<T> cannot be applied to given types;
          assertThat(singleton.isSet(singletonId)).isFalse();
                              ^
    required: no arguments
    found:    String
    reason: actual and formal argument lists differ in length
    where T is a type-variable:
      T extends Object declared in interface Singleton
  7 errors

* Try:
> Check your code and dependencies to fix the compilation error(s)
> Run with --scan to generate a Build Scan (Powered by Develocity).

BUILD FAILED in 1s

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.
10 actionable tasks: 1 executed, 9 up-to-date

[Incubating] Problems report is available at: file:///home/runner/work/graalvm-reachability-metadata/graalvm-reachability-metadata/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':compileTestJava'.
> Compilation failed

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to generate a Build Scan (Powered by Develocity).
> Get more help at https://help.gradle.org.

Deprecated Gradle features were used in this build, making it incompatible with Gradle 10.

You can use '--warning-mode all' to show the individual deprecation warnings and determine if they come from your own scripts or plugins.

For more on this, please refer to https://docs.gradle.org/9.1.0/userguide/command_line_interface.html#sec:command_line_warnings in the Gradle documentation.

BUILD FAILED in 3s
11 actionable tasks: 3 executed, 8 up-to-date
❌ ERROR: 'javac compile' for 2.0.Final failed with exit code 1.
FAILED[javac compile][2.0.Final][GVM_TCK_NATIVE_IMAGE_MODE="future-defaults-all" GVM_TCK_LV="2.0.Final" ./gradlew clean compileTestJava -Pcoordinates="org.jboss.weld:weld-spi:1.2.0.Beta1"]
```

--- a/metadata/org.jboss.weld/weld-spi/2.0.Final/reachability-metadata.json
+++ b/metadata/org.jboss.weld/weld-spi/2.0.Final/reachability-metadata.json
@@ -1,0 +1,16 @@
+{
+  "reflection": [
+    {
+      "condition": {
+        "typeReached": "org.jboss.weld.bootstrap.api.SingletonProvider"
+      },
+      "type": "org.jboss.weld.bootstrap.api.helpers.IsolatedStaticSingletonProvider",
+      "methods": [
+        {
+          "name": "<init>",
+          "parameterTypes": []
+        }
+      ]
+    }
+  ]
+}

--- a/metadata/org.jboss.weld/weld-spi/index.json
+++ b/metadata/org.jboss.weld/weld-spi/index.json
@@ -1,6 +1,15 @@
 [
   {
     "latest" : true,
+    "metadata-version" : "2.0.Final",
+    "tested-versions" : [
+      "2.0.Final"
+    ],
+    "allowed-packages" : [
+      "org.jboss.weld"
+    ]
+  },
+  {
     "metadata-version" : "1.2.0.Beta1",
     "source-code-url" : "https://repo.maven.apache.org/maven2/org/jboss/weld/weld-spi/$version$/weld-spi-$version$-sources.jar",
     "repository-url" : "https://github.com/weld/api",

--- a/metadata/org.jboss.weld/weld-spi/index.json
+++ b/metadata/org.jboss.weld/weld-spi/index.json
@@ -2,6 +2,11 @@
   {
     "latest" : true,
     "metadata-version" : "2.0.Final",
+    "source-code-url" : "https://repo.maven.apache.org/maven2/org/jboss/weld/weld-spi/$version$/weld-spi-$version$-sources.jar",
+    "repository-url" : "https://github.com/weld/api",
+    "test-code-url" : "https://github.com/weld/api/tree/$version$/weld-spi/src/test/java",
+    "documentation-url" : "https://repo.maven.apache.org/maven2/org/jboss/weld/weld-spi/$version$/weld-spi-$version$-javadoc.jar",
+    "description" : "Weld SPI provides the service-provider interfaces used to integrate Weld, the CDI reference implementation, with Java EE containers and runtime services. It defines bootstrap, deployment, resource, transaction, security, validation, EJB, injection, servlet, and serialization contracts for container integration.",
     "tested-versions" : [
       "2.0.Final"
     ],

--- a/stats/org.jboss.weld/weld-spi/2.0.Final/execution-metrics.json
+++ b/stats/org.jboss.weld/weld-spi/2.0.Final/execution-metrics.json
@@ -1,0 +1,99 @@
+{
+  "fix_javac_fail:2026-05-02": {
+    "timestamp": "2026-05-02T21:45:29.385266Z",
+    "library": "org.jboss.weld:weld-spi:2.0.Final",
+    "starting_commit": "430ea59423dfed73e7a23d3741171d8add6077c9",
+    "ending_commit": "b74074fc42e29151c5c89aa25e4c57bd4d0cd2b4",
+    "previous_library": "org.jboss.weld:weld-spi:1.2.0.Beta1",
+    "strategy_name": "javac_iterative_with_coverage_sources_pi_gpt-5.5",
+    "agent": "pi",
+    "model": "oca/gpt-5.5",
+    "status": "success",
+    "stats": {
+      "version": "2.0.Final",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 1,
+        "totalCalls": 1
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 67,
+          "missed": 1328,
+          "ratio": 0.048029,
+          "total": 1395
+        },
+        "line": {
+          "covered": 26,
+          "missed": 289,
+          "ratio": 0.08254,
+          "total": 315
+        },
+        "method": {
+          "covered": 11,
+          "missed": 174,
+          "ratio": 0.059459,
+          "total": 185
+        }
+      }
+    },
+    "previous_library_stats": {
+      "version": "1.2.0.Beta1",
+      "dynamicAccess": {
+        "breakdown": {
+          "reflection": {
+            "coverageRatio": 1.0,
+            "coveredCalls": 1,
+            "totalCalls": 1
+          }
+        },
+        "coverageRatio": 1.0,
+        "coveredCalls": 1,
+        "totalCalls": 1
+      },
+      "libraryCoverage": {
+        "instruction": {
+          "covered": 67,
+          "missed": 1127,
+          "ratio": 0.056114,
+          "total": 1194
+        },
+        "line": {
+          "covered": 26,
+          "missed": 246,
+          "ratio": 0.095588,
+          "total": 272
+        },
+        "method": {
+          "covered": 11,
+          "missed": 152,
+          "ratio": 0.067485,
+          "total": 163
+        }
+      }
+    },
+    "metrics": {
+      "input_tokens_used": 15119,
+      "cached_input_tokens_used": 65536,
+      "output_tokens_used": 1162,
+      "iterations": 1,
+      "cost_usd": 0.0677,
+      "tested_library_loc": 26,
+      "metadata_entries": 2,
+      "previous_library_metadata_entries": 2,
+      "code_coverage_percent": 8.25,
+      "previous_library_coverage_percent": 9.56
+    },
+    "artifacts": {
+      "test_file": "tests/src/org.jboss.weld/weld-spi/2.0.Final/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java",
+      "metadata_file": "metadata/org.jboss.weld/weld-spi/2.0.Final/reachability-metadata.json"
+    }
+  }
+}

--- a/stats/org.jboss.weld/weld-spi/2.0.Final/stats.json
+++ b/stats/org.jboss.weld/weld-spi/2.0.Final/stats.json
@@ -1,0 +1,37 @@
+{
+  "versions" : [ {
+    "version" : "2.0.Final",
+    "dynamicAccess" : {
+      "breakdown" : {
+        "reflection" : {
+          "coverageRatio" : 1.0,
+          "coveredCalls" : 1,
+          "totalCalls" : 1
+        }
+      },
+      "coverageRatio" : 1.0,
+      "coveredCalls" : 1,
+      "totalCalls" : 1
+    },
+    "libraryCoverage" : {
+      "instruction" : {
+        "covered" : 67,
+        "missed" : 1328,
+        "ratio" : 0.048029,
+        "total" : 1395
+      },
+      "line" : {
+        "covered" : 26,
+        "missed" : 289,
+        "ratio" : 0.08254,
+        "total" : 315
+      },
+      "method" : {
+        "covered" : 11,
+        "missed" : 174,
+        "ratio" : 0.059459,
+        "total" : 185
+      }
+    }
+  } ]
+}

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/.gitignore
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/.gitignore
@@ -1,0 +1,4 @@
+gradlew.bat
+gradlew
+gradle/
+build/

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/build.gradle
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/build.gradle
@@ -1,0 +1,27 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+plugins {
+    id "org.graalvm.internal.tck"
+}
+
+String libraryVersion = tck.testedLibraryVersion.get()
+
+dependencies {
+    testImplementation "org.jboss.weld:weld-spi:$libraryVersion"
+    testImplementation 'org.assertj:assertj-core:3.22.0'
+}
+
+graalvmNative {
+    agent {
+        defaultMode = "conditional"
+        modes {
+            conditional {
+                userCodeFilterPath = "user-code-filter.json"
+            }
+        }
+    }
+}

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/gradle.properties
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/gradle.properties
@@ -1,0 +1,6 @@
+# Optional explicit tested library coordinates (format: group:artifact:version).
+# If omitted, fallback resolution in org.graalvm.internal.tck.gradle still uses
+# environment variable GVM_TCK_LC and then this property.
+library.coordinates = org.jboss.weld:weld-spi:2.0.Final
+library.version = 2.0.Final
+metadata.dir = org.jboss.weld/weld-spi/2.0.Final/

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/settings.gradle
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/settings.gradle
@@ -1,0 +1,20 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+pluginManagement {
+    def tckPath = Objects.requireNonNullElse(
+            System.getenv("GVM_TCK_TCKDIR"),
+            "../../../../tck-build-logic"
+    )
+    includeBuild(tckPath)
+    repositories { mavenLocal(); gradlePluginPortal() }
+}
+
+plugins {
+    id "org.graalvm.internal.tck-settings" version "1.0.0-SNAPSHOT"
+}
+
+rootProject.name = 'org.jboss.weld.weld-spi_tests'

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java
@@ -1,0 +1,50 @@
+/*
+ * Copyright and related rights waived via CC0
+ *
+ * You should have received a copy of the CC0 legalcode along with this
+ * work. If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
+ */
+package org_jboss_weld.weld_spi;
+
+import org.jboss.weld.bootstrap.api.Singleton;
+import org.jboss.weld.bootstrap.api.SingletonProvider;
+import org.jboss.weld.bootstrap.api.helpers.IsolatedStaticSingletonProvider;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+public class SingletonProviderTest {
+    @BeforeEach
+    void resetBeforeTest() {
+        SingletonProvider.reset();
+    }
+
+    @AfterEach
+    void resetAfterTest() {
+        SingletonProvider.reset();
+    }
+
+    @Test
+    void instanceInitializesDefaultIsolatedStaticProvider() {
+        SingletonProvider provider = SingletonProvider.instance();
+
+        assertThat(provider).isInstanceOf(IsolatedStaticSingletonProvider.class);
+
+        Singleton<String> singleton = provider.create(String.class);
+        String singletonId = "test-application";
+        assertThat(singleton.isSet(singletonId)).isFalse();
+        assertThatThrownBy(() -> singleton.get(singletonId))
+                .isInstanceOf(IllegalStateException.class)
+                .hasMessageContaining("Singleton is not set");
+
+        singleton.set(singletonId, "stored value");
+        assertThat(singleton.isSet(singletonId)).isTrue();
+        assertThat(singleton.get(singletonId)).isEqualTo("stored value");
+
+        singleton.clear(singletonId);
+        assertThat(singleton.isSet(singletonId)).isFalse();
+    }
+}

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/src/test/java/org_jboss_weld/weld_spi/SingletonProviderTest.java
@@ -34,17 +34,16 @@ public class SingletonProviderTest {
         assertThat(provider).isInstanceOf(IsolatedStaticSingletonProvider.class);
 
         Singleton<String> singleton = provider.create(String.class);
-        String singletonId = "test-application";
-        assertThat(singleton.isSet(singletonId)).isFalse();
-        assertThatThrownBy(() -> singleton.get(singletonId))
+        assertThat(singleton.isSet()).isFalse();
+        assertThatThrownBy(singleton::get)
                 .isInstanceOf(IllegalStateException.class)
                 .hasMessageContaining("Singleton is not set");
 
-        singleton.set(singletonId, "stored value");
-        assertThat(singleton.isSet(singletonId)).isTrue();
-        assertThat(singleton.get(singletonId)).isEqualTo("stored value");
+        singleton.set("stored value");
+        assertThat(singleton.isSet()).isTrue();
+        assertThat(singleton.get()).isEqualTo("stored value");
 
-        singleton.clear(singletonId);
-        assertThat(singleton.isSet(singletonId)).isFalse();
+        singleton.clear();
+        assertThat(singleton.isSet()).isFalse();
     }
 }

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-jupiter.xml
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-jupiter.xml
@@ -1,0 +1,64 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="JUnit Jupiter" tests="1" skipped="0" failures="0" errors="0" time="0" hostname="kung-fu-workstation" timestamp="2026-05-02T16:01:37">
+<properties>
+<property name="file.encoding" value="UTF-8"/>
+<property name="file.separator" value="/"/>
+<property name="java.class.path" value=""/>
+<property name="java.class.version" value="69.0"/>
+<property name="java.endorsed.dirs" value=""/>
+<property name="java.ext.dirs" value=""/>
+<property name="java.io.tmpdir" value="/tmp"/>
+<property name="java.library.path" value="/usr/lib64:/lib64:/lib:/usr/lib"/>
+<property name="java.runtime.name" value="GraalVM Runtime Environment"/>
+<property name="java.runtime.version" value="25.0.3+9-LTS-jvmci-b01"/>
+<property name="java.specification.name" value="Java Platform API Specification"/>
+<property name="java.specification.vendor" value="Oracle Corporation"/>
+<property name="java.specification.version" value="25"/>
+<property name="java.vendor" value="Oracle Corporation"/>
+<property name="java.vendor.url" value="https://www.graalvm.org/"/>
+<property name="java.vendor.version" value="Oracle GraalVM 25.0.3+9.1"/>
+<property name="java.version" value="25.0.3"/>
+<property name="java.version.date" value="2026-04-21"/>
+<property name="java.vm.info" value="serial gc, compressed references"/>
+<property name="java.vm.name" value="Substrate VM"/>
+<property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
+<property name="java.vm.specification.vendor" value="Oracle Corporation"/>
+<property name="java.vm.specification.version" value="25"/>
+<property name="java.vm.vendor" value="Oracle Corporation"/>
+<property name="java.vm.version" value="25.0.3+9-LTS"/>
+<property name="jdk.lang.Process.launchMechanism" value="FORK"/>
+<property name="junit.jupiter.execution.timeout.default" value="60 s"/>
+<property name="junit.jupiter.execution.timeout.mode" value="enabled"/>
+<property name="junit.jupiter.execution.timeout.thread.mode.default" value="separate_thread"/>
+<property name="line.separator" value="
+"/>
+<property name="native.encoding" value="UTF-8"/>
+<property name="org.graalvm.nativeimage.imagecode" value="runtime"/>
+<property name="org.graalvm.nativeimage.kind" value="executable"/>
+<property name="os.arch" value="amd64"/>
+<property name="os.name" value="Linux"/>
+<property name="os.version" value="6.17.0-22-generic"/>
+<property name="path.separator" value=":"/>
+<property name="stderr.encoding" value="UTF-8"/>
+<property name="stdin.encoding" value="UTF-8"/>
+<property name="stdout.encoding" value="UTF-8"/>
+<property name="sun.arch.data.model" value="64"/>
+<property name="sun.jnu.encoding" value="UTF-8"/>
+<property name="user.country" value="US"/>
+<property name="user.dir" value="/home/vjovanov/c/forge/forge2/graalvm-reachability-metadata/forge/local_repositories/forge_worktrees/4315-ac6368d2/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1"/>
+<property name="user.home" value="/home/vjovanov"/>
+<property name="user.language" value="en"/>
+<property name="user.name" value="vjovanov"/>
+<property name="user.timezone" value="Europe/Zurich"/>
+</properties>
+<testcase name="instanceInitializesDefaultIsolatedStaticProvider()" classname="org_jboss_weld.weld_spi.SingletonProviderTest" time="0">
+<system-out><![CDATA[
+unique-id: [engine:junit-jupiter]/[class:org_jboss_weld.weld_spi.SingletonProviderTest]/[method:instanceInitializesDefaultIsolatedStaticProvider()]
+display-name: instanceInitializesDefaultIsolatedStaticProvider()
+]]></system-out>
+</testcase>
+<system-out><![CDATA[
+unique-id: [engine:junit-jupiter]
+display-name: JUnit Jupiter
+]]></system-out>
+</testsuite>

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-jupiter.xml
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-jupiter.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuite name="JUnit Jupiter" tests="1" skipped="0" failures="0" errors="0" time="0" hostname="kung-fu-workstation" timestamp="2026-05-02T16:01:37">
+<testsuite name="JUnit Jupiter" tests="1" skipped="0" failures="0" errors="0" time="0" hostname="kung-fu-workstation" timestamp="2026-05-02T23:44:18">
 <properties>
 <property name="file.encoding" value="UTF-8"/>
 <property name="file.separator" value="/"/>
@@ -45,7 +45,7 @@
 <property name="sun.arch.data.model" value="64"/>
 <property name="sun.jnu.encoding" value="UTF-8"/>
 <property name="user.country" value="US"/>
-<property name="user.dir" value="/home/vjovanov/c/forge/forge2/graalvm-reachability-metadata/forge/local_repositories/forge_worktrees/4315-ac6368d2/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1"/>
+<property name="user.dir" value="/home/vjovanov/c/forge/forge10/graalvm-reachability-metadata/forge/local_repositories/forge_worktrees/4427-d56562a9/tests/src/org.jboss.weld/weld-spi/2.0.Final"/>
 <property name="user.home" value="/home/vjovanov"/>
 <property name="user.language" value="en"/>
 <property name="user.name" value="vjovanov"/>

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-vintage.xml
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-vintage.xml
@@ -1,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<testsuite name="JUnit Vintage" tests="0" skipped="0" failures="0" errors="0" time="0" hostname="kung-fu-workstation" timestamp="2026-05-02T16:01:37">
+<properties>
+<property name="file.encoding" value="UTF-8"/>
+<property name="file.separator" value="/"/>
+<property name="java.class.path" value=""/>
+<property name="java.class.version" value="69.0"/>
+<property name="java.endorsed.dirs" value=""/>
+<property name="java.ext.dirs" value=""/>
+<property name="java.io.tmpdir" value="/tmp"/>
+<property name="java.library.path" value="/usr/lib64:/lib64:/lib:/usr/lib"/>
+<property name="java.runtime.name" value="GraalVM Runtime Environment"/>
+<property name="java.runtime.version" value="25.0.3+9-LTS-jvmci-b01"/>
+<property name="java.specification.name" value="Java Platform API Specification"/>
+<property name="java.specification.vendor" value="Oracle Corporation"/>
+<property name="java.specification.version" value="25"/>
+<property name="java.vendor" value="Oracle Corporation"/>
+<property name="java.vendor.url" value="https://www.graalvm.org/"/>
+<property name="java.vendor.version" value="Oracle GraalVM 25.0.3+9.1"/>
+<property name="java.version" value="25.0.3"/>
+<property name="java.version.date" value="2026-04-21"/>
+<property name="java.vm.info" value="serial gc, compressed references"/>
+<property name="java.vm.name" value="Substrate VM"/>
+<property name="java.vm.specification.name" value="Java Virtual Machine Specification"/>
+<property name="java.vm.specification.vendor" value="Oracle Corporation"/>
+<property name="java.vm.specification.version" value="25"/>
+<property name="java.vm.vendor" value="Oracle Corporation"/>
+<property name="java.vm.version" value="25.0.3+9-LTS"/>
+<property name="jdk.lang.Process.launchMechanism" value="FORK"/>
+<property name="junit.jupiter.execution.timeout.default" value="60 s"/>
+<property name="junit.jupiter.execution.timeout.mode" value="enabled"/>
+<property name="junit.jupiter.execution.timeout.thread.mode.default" value="separate_thread"/>
+<property name="line.separator" value="
+"/>
+<property name="native.encoding" value="UTF-8"/>
+<property name="org.graalvm.nativeimage.imagecode" value="runtime"/>
+<property name="org.graalvm.nativeimage.kind" value="executable"/>
+<property name="os.arch" value="amd64"/>
+<property name="os.name" value="Linux"/>
+<property name="os.version" value="6.17.0-22-generic"/>
+<property name="path.separator" value=":"/>
+<property name="stderr.encoding" value="UTF-8"/>
+<property name="stdin.encoding" value="UTF-8"/>
+<property name="stdout.encoding" value="UTF-8"/>
+<property name="sun.arch.data.model" value="64"/>
+<property name="sun.jnu.encoding" value="UTF-8"/>
+<property name="user.country" value="US"/>
+<property name="user.dir" value="/home/vjovanov/c/forge/forge2/graalvm-reachability-metadata/forge/local_repositories/forge_worktrees/4315-ac6368d2/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1"/>
+<property name="user.home" value="/home/vjovanov"/>
+<property name="user.language" value="en"/>
+<property name="user.name" value="vjovanov"/>
+<property name="user.timezone" value="Europe/Zurich"/>
+</properties>
+<system-out><![CDATA[
+unique-id: [engine:junit-vintage]
+display-name: JUnit Vintage
+]]></system-out>
+</testsuite>

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-vintage.xml
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/test-results-native/test/TEST-junit-vintage.xml
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuite name="JUnit Vintage" tests="0" skipped="0" failures="0" errors="0" time="0" hostname="kung-fu-workstation" timestamp="2026-05-02T16:01:37">
+<testsuite name="JUnit Vintage" tests="0" skipped="0" failures="0" errors="0" time="0" hostname="kung-fu-workstation" timestamp="2026-05-02T23:44:18">
 <properties>
 <property name="file.encoding" value="UTF-8"/>
 <property name="file.separator" value="/"/>
@@ -45,7 +45,7 @@
 <property name="sun.arch.data.model" value="64"/>
 <property name="sun.jnu.encoding" value="UTF-8"/>
 <property name="user.country" value="US"/>
-<property name="user.dir" value="/home/vjovanov/c/forge/forge2/graalvm-reachability-metadata/forge/local_repositories/forge_worktrees/4315-ac6368d2/tests/src/org.jboss.weld/weld-spi/1.2.0.Beta1"/>
+<property name="user.dir" value="/home/vjovanov/c/forge/forge10/graalvm-reachability-metadata/forge/local_repositories/forge_worktrees/4427-d56562a9/tests/src/org.jboss.weld/weld-spi/2.0.Final"/>
 <property name="user.home" value="/home/vjovanov"/>
 <property name="user.language" value="en"/>
 <property name="user.name" value="vjovanov"/>

--- a/tests/src/org.jboss.weld/weld-spi/2.0.Final/user-code-filter.json
+++ b/tests/src/org.jboss.weld/weld-spi/2.0.Final/user-code-filter.json
@@ -1,0 +1,10 @@
+{
+  "rules" : [
+    {
+      "excludeClasses" : "**"
+    },
+    {
+      "includeClasses" : "org.jboss.weld.**"
+    }
+  ]
+}

